.github/workflows/ci.yml

```diff
@@ -575,7 +575,10 @@ jobs:
           echo "deb [signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
           sudo sed -i 's/^# deb-src/deb-src/' /etc/apt/sources.list
           sudo apt-get update
-          sudo apt-get install -y libvirt-daemon libvirt-daemon-system vagrant
+          # Pinned to 2.4.1-1 until https://github.com/hashicorp/vagrant/pull/13532 in released version
+          sudo apt-get install -y libvirt-daemon libvirt-daemon-system vagrant=2.4.1-1 ovmf
+          # https://github.com/vagrant-libvirt/vagrant-libvirt/issues/1725#issuecomment-1454058646
+          sudo cp /usr/share/OVMF/OVMF_VARS_4M.fd /var/lib/libvirt/qemu/nvram/
```

Comment:
Can you also add that this change is cherry-picked from https://github.com/containerd/containerd/commit/1db0064c6af91c6b94c5bd0241d33b6cd0015478

Reply:
officially through `git cherry-pick -x` or you think it needs a comment?

Reply:
I prefer the `cherry-pick -x`. 

I think its still working fine without the other changes (setting nvram in vagrant file) in the above mentioned commit because we are still on ubuntu-22.04 in release branch. But once we upgrade we are anyway going to need those changes.